spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -208,7 +208,7 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
       expr match {
         case agg: AggregateExpression =>
           agg.aggregateFunction match {
-            case _: Min | _: Max | _: Count =>
```

Comment:
This disables `count` for both aggregation and window?

Reply:
Oh, nvm, I checked the source code. This is only for window functions.

Reply:
Can we add a comment here?

Reply:
Added